app.py

```
@@ -299,7 +301,37 @@ def favicon():
 def index():
-    return render_template("index.html")
+    # Get statistics for homepage
+    total_public_rooms = Room.query.filter_by(is_private=False, is_archived=False).count()
+    total_private_rooms = 0
+    user = auth.get_current_user()
+    if user:
+        total_private_rooms = Room.query.filter_by(is_private=True, is_archived=False, owner_id=user.id).count()
+
+    # Active rooms (with messages in last 24 hours)
+    from datetime import datetime, timedelta
+    twenty_four_hours_ago = datetime.utcnow() - timedelta(hours=24)
```

Ruff (0.14.4)

Local variable `twenty_four_hours_ago` is assigned to but never used

Remove assignment to unused variable `twenty_four_hours_ago`

(F841)